modules/ROOT/pages/mule-app-dev.adoc

```diff
@@ -0,0 +1,184 @@
+
+Note that you can also create your own source using the
+xref:mule-sdk::index.adoc[Mule SDK].
+
+=== Processors
+
+Once a flow is triggered through a source component, subsequent components process
+the data as it travels through the flow. By default, each processor
+that receives a Mule event returns a new Mule message, typically with a set
+of attributes and the message payload that the processor returns. The processor
+passes on the new message as output to the next processor in the flow.
+
+Processors available to Mule applications include:
+
+* Components from modules and connectors. Examples include operations that read
+from and write to an external resource and that validate data in the Mule application.
++
+Some operations can make client requests to external resources and services
+(including external databases and systems, such as Salesforce, Workday,
+ServiceNow, and many others) and to other Mule applications. Others can run your
+custom code, support OAuth configurations, and handle communication through
+asynchronous queues, for example. A variety of other operations are
+available.
++
+* Core components can route data, perform data transformations, handle errors
+that might occur when processing the event, and perform other tasks in a
+Mule application.
++
+Transformers (such as the Transform Message, Set Variable, and others) are
+key to exchanging data between nodes. Transformers enable Mule to convert
+message data in the Mule event to a format that another application or service can
+read.
++
+Mule also enables content enrichment of messages (through
+xref:target-variables.adoc[Target Parameters]) so that you can retrieve
+additional data and attach it to the message.
+
+Note that you can also create your own Event processors using the
+xref:mule-sdk::index.adoc[Mule SDK].
+
+== Security
+
+You can provide security to your Mule applications by encrypting properties, configuring
+secure communications over TLS, setting up authentication over OAuth 2.0, and
+providing cryptographic and other capabilities, such as FIPS compliance.
+
+* xref:securing.adoc[Security]
+
+== Development Environments
+
+You can develop a Mule application using
+xref:docs-studio::index.adoc[Anypoint Studio] (an Eclipse-based IDE),
+xref:docs-design-center::about-designing-a-mule-application.adoc[Flow Designer]
+(a Cloud-based application in Design Center, on Anypoint Platform),
+or, if you are an advanced developer, in your own IDE.
+
+For example, in Studio, you build and design a Mule application in a project
+that contains one or more an XML-based files. A Mule project supports all the
+dependencies required for development. The
+xref:7.3@docs-studio::index.adoc#package-explorer[Package Explorer] in Studio
+provides access to the project folders and files that make up a Mule project
+support a design-time environment in which you can also build, run, and test
+your Mule application. Flow Designer supports a Cloud-based version of a Mule project.
```

Comment:
GLOBALLY change "Cloud-based" to "cloud-based." 
We now lowercase "cloud" in the same way that we have come to lowercase "internet."